### ISSUE TITLE
Can't run from the command line

### ISSUE THREAD

**Issue by scolyer** [goldibex/targaryen#15]
Just tried to npm install on OSX El Capitan -  targaryen won't run from the command line (command not found). Any tips on how to resolve?

**Comment by goldibex** [comment]
Please supply your node and npm versions as well as the complete log of the
problematic command line session.

**Comment by scolyer** [comment]
node v0.12.5
npm 2.11.2

```
Seans-MacBook-Pro:koleido-ember sean$ npm install -g targaryen
targaryen@1.1.4 /usr/local/lib/node_modules/targaryen
├── replaceall@0.1.6
├── extend@2.0.1
├── colors@1.1.2
├── minimist@1.2.0
├── cli-table@0.3.1 (colors@1.0.3)
└── esprima@1.2.5
Seans-MacBook-Pro:koleido-ember sean$ targaryen
-bash: targaryen: command not found
```

**Comment by weactivist** [comment]
I have the same problem on Yosemite:

    $ node -v
    v4.2.0
    $ npm -v
    3.3.6

    $ npm install -g targaryen
    /Users/hmmrsr/.node/lib
    └─┬ targaryen@1.1.4 
      ├─┬ cli-table@0.3.1 
      │ └── colors@1.0.3 
      ├── colors@1.1.2 
      ├── esprima@1.2.5 
      ├── extend@2.0.1 
      ├── minimist@1.2.0 
      └── replaceall@0.1.6

    $ targaryen
    -bash: /Users/hmmrsr/.node/bin/targaryen: No such file or directory

Installing Gulp, or other packages, works without issues.

**Comment by goldibex** [comment]
Hi guys, working this problem now, should have a fix shortly

**Comment by daguej** [comment]
What is your `PATH`?  (`echo $PATH`)

**Comment by goldibex** [comment]
Fixed by #16.

**Comment by scolyer** [comment]
Awesome, thanks!
